### PR TITLE
TCD-2622 and TCD-2372: Remove references to SC v4.6.x and add 4.8.0 CLI flags

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by LoryApiFortress** [comment]
@lysannep please review. Thanks

**Comment by LoryApiFortress** [comment]
Integrated TCD-2372 in the PR. Marked as a draft because I'm waiting on feedback from Dan

File: docs/dev/cli/sauce-connect-proxy.md [hunk 1]
```diff
@@ -167,57 +191,45 @@ HTTP Header Injection is disabled for all HTTPS domains passed to the `--no-ssl-
...
-<p><small>| OPTIONAL | STRING | <span className="sauceRed">4.6.x</span> <span className="sauceRed">4.7.x</span> <span className="sauceRed">4.8.x</span>| </small></p>
-
-**Description**: Sets the URL for the [data center endpoint](/basics/data-center-endpoints) of the location where the device you're testing on is hosted.<br/>
-**Environment variable**: `SAUCE_REST_URL`<br/>
-**Shorthand**: `-x`
-
-```bash
-#US-West-1
-./sc -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -x https://api.us-west-1.saucelabs.com/rest/v1
+### `--no-remove-colliding-tunnels`
```
Comment (lysannep) [inline docs/dev/cli/sauce-connect-proxy.md:202]: We should also remove the descriptions for the CLI flags that are only available in version 4.6.x (including this one (`--no-remove-colliding-tunnels`)).

Screenshot 2023-05-05 at 7 06 20 PM

File: docs/dev/cli/sauce-connect-proxy.md [hunk 2]
```diff
@@ -167,57 +191,45 @@ HTTP Header Injection is disabled for all HTTPS domains passed to the `--no-ssl-
...
 
 :::note
-Effective with version 4.7.0, this flag was deprecated and replaced by [`--region`](/dev/cli/sauce-connect-proxy/#--region).
+Effective with version 4.7.0, this flag was deprecated and replaced by [`--tunnel-pool`](#--tunnel-pool).
 :::
 
 ---
 
-### `--no-remove-colliding-tunnels`
+### `--extra-info`
```
Comment (lysannep) [inline docs/dev/cli/sauce-connect-proxy.md:216]: The section for `--extra-info `here should be removed:
Screenshot 2023-05-05 at 7 12 10 PM

File: docs/dev/cli/sauce-connect-proxy.md [hunk 3]
```diff
@@ -447,7 +469,7 @@ This flag will only send the header Authorization with a type of "Basic." If a r
 
 ### `--tunnel-capath`
```
Comment (lysannep) [inline docs/dev/cli/sauce-connect-proxy.md:470]: The sections for `--tunnel-capath` and `--capath` should be removed as well:
Screenshot 2023-05-05 at 7 21 45 PM

**Review by lysannep — COMMENTED** [review]
@LoryApiFortress I added a few comments. Thank you!

**Review by lysannep — APPROVED** [review]

**Comment by LoryApiFortress** [comment]
I'm still waiting for feedback ([TCD-2372](https://saucedev.atlassian.net/browse/TCD-2372)) (same question also asked in [SC-4206](https://saucedev.atlassian.net/browse/SC-4206)) but I can open a new PR once I get the feedback

**Comment by LoryApiFortress** [comment]
@waggledans can you please review? Thanks

File: docs/secure-connections/sauce-connect/security-authentication.md [hunk 4]
```diff
@@ -151,7 +151,7 @@ On macOS machines, certificates are pre-installed as part of the [Trust Store](h
 
 Sauce Connect Proxy reverses tunnel VM-to-test target traffic through the TLS connection from Sauce Connect-to-tunnel endpoints. Your Selenium and Appium webdriver traffic is sent over `http(80)` or `https(443)` to `ondemand.saucelabs.com`, which has its own TLS certificate that's then passed to the test VM.
```
Comment (waggledans) [inline docs/secure-connections/sauce-connect/security-authentication.md:152]:
```suggestion
Sauce Connect Proxy routes VM-to-test target traffic through the TLS connection between the Sauce Connect Proxy client and the Sauce Connect Server. Sauce Connect Proxy is not used with your Selenium and Appium webdriver traffic to the Sauce Labs `on-demand` endpoint, for example, `ondemand.us-west-1.saucelabs.com`.
```

File: docs/secure-connections/sauce-connect/security-authentication.md [hunk 5]
```diff
@@ -117,7 +117,7 @@ When securing Sauce Connect Proxy, be sure to allowlist these sites so that the
 
 - **OCSP:** http://gp.symcd.com
 - **OCSP Servers for VDC/RDC clouds:** http://ocsp.digicert.com, http://status.geotrust.com
-- **OCSP Servers for Headless:** http://ocsp.int-x3.letsencrypt.org,  http://isrg.trustid.ocsp.identrust.com
+- **OCSP Servers for Headless:** http://ocsp.int-x3.letsencrypt.org, http://isrg.trustid.ocsp.identrust.com
```
Comment (waggledans) [inline docs/secure-connections/sauce-connect/security-authentication.md:120]:
```suggestion
```

File: docs/secure-connections/sauce-connect/security-authentication.md [hunk 6]
```diff
@@ -117,7 +117,7 @@ When securing Sauce Connect Proxy, be sure to allowlist these sites so that the
 
 - **OCSP:** http://gp.symcd.com
 - **OCSP Servers for VDC/RDC clouds:** http://ocsp.digicert.com, http://status.geotrust.com
```
Comment (waggledans) [inline docs/secure-connections/sauce-connect/security-authentication.md:119]:
```suggestion
```

**Comment by LoryApiFortress** [comment]
@waggledans I applied all your suggestions, are you ok with merging this to `main`?

**Comment by LoryApiFortress** [comment]
@waggledans I applied all your suggestions, are you ok with merging this to `main`?

**Review by waggledans — APPROVED** [review]

**Pull request merged**